internal/driver/glfw/window_desktop.go

```diff
@@ -399,6 +402,14 @@ func (w *window) mouseScrolled(viewport *glfw.Window, xoff float64, yoff float64
 	w.processMouseScrolled(xoff, yoff)
 }
 
+func (w *window) lastWalked() time.Time {
+	return w.lastWalkedTime
+}
```

Comment:
Do we really need this as a function if it only is internal field?

Reply:
Probably not, I did it as a function to protect it from accidental changes from another package (basically to make it read-only), but since it's all internal code anyway I guess it's premature access control

Reply:
Yeah, I think there are many other internal fields that we don't have getters for so it seems more consistent to not add one more. I think we might also want to remove `.windows()` from the driver now that we don't have the locking there etc.

Reply:
Did you mark this as resolved by accident? I don't see it being fixed